### DISCUSSION TITLE
Who's hiring (Nov 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#42311] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #42310.

*If you're looking for last month's discussion, see #41663. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!*

**Comment by jaxomlotus** [comment]
Burner (Generative AI websites) | Full Stack Developer, ML & Design | Full Time | Remote (EST timezone based) | $115-155k base + equity

Burner is on a mission to use AI to build and continually optimize webpages, so people can focus on their actual products.

Burner (stealth) uses Machine Learning, Artificial Intelligence and Computer Vision to analyze and categorize page content, generate variants and manage experiments to optimize page performance.

Burner is hiring front end engineers with full stack experience. Looking for Node, Next.js/React and MySQL experience. We are also looking for machine learning engineers and designers.

https://new.burnerpage.com/jobs/3/front-end-engineer

We are also looking for Machine Learning engineers and a great UX designer (other jobs are here: https://new.burnerpage.com/jobs/

**Reply by zhyd1997** [reply]
Hi, I am interested in this opportunity:

https://github.com/vercel/next.js/discussions/42310#discussioncomment-4030281

But I have no MySQL experience.

**Reply by rajibola** [reply]
Hi @jaxomlotus, there's no section for the application in the link you shared

**Reply by AsherAshrafM** [reply]
Hi @jaxomlotus, I am very interested in the job. I have experience in Node, Next.js/React and MySQL. I have been building with Node for 7 years and MySQL for 16 years since 2006. Please see my portfolio. Thank you. 🙏

https://sachu-developer.webflow.io/

**Reply by ryokochang** [reply]
Hi, I'm very interested in the job. Here is linked profile https://www.linkedin.com/in/ryokochang/

**Comment by jaxomlotus** [comment]
The contact email is on the bottom of the job description

**Comment by Joseph-Lion** [comment]
**Forty Eight Point One | Developer | London & Remote | Full-time | £40-45k Base** 

We are looking for experienced developers to join our team. We work with brands to help them solve their biggest digital challenges. Every project presents an entirely different challenge – and you will work on everything from large team development projects to smaller independent builds. From platforms to e-commerce sites to apps. Regardless of the size, we place the same importance on our approach, languages and the frameworks we use.

We are not afraid to use new technologies and thrive on pushing the boundaries for our clients. We have a talented team that continues to design and develop award-winning digital products. We’re keen to expand the team as we move into larger, more ambitious projects.

Feel free to email me for more information Joe@fortyeight.one

[Job Spec](https://fortyeight.one/middleweight-front-end-developer/)
[Our Website](https://fortyeight.one/)

**Reply by AsherAshrafM** [reply]
Hi @Joseph-Lion , I am interested in the position. Please see my portfolio. Thank you. 🙏

https://sachu-developer.webflow.io/

**Reply by vigneshshiv** [reply]
Hi @Joseph-Lion, I'm interested, Discussion - [#42310](https://github.com/vercel/next.js/discussions/42310#discussioncomment-4065075)
Please check out my profile - https://github.com/vigneshshiv

**Comment by nwidynski** [comment]
<b>Equity Bytes Holding | Engineering,Data,Design & Recruiting | Full-time 60-100k+€ | Berlin (On-Site/Hybrid)</b>


We're on track to build the future of collectibles and social commerce. Our portfolio consists of multiple brands that horizontally integrate & scale together.

Projects include marketplace infrastructure, a social commerce platform & a VC-backed alternative asset fund.

Being entirely bootstrapped, we're embracing the idea of a lean company with flat hierarchy. We are looking for highly talented specialists as well as any experienced generalists.

You will have the chance to earn considerable equity in what we believe will be the next Ebay.

Stack: Typescript / Next.js / Stripe Connect
Roles:
* (Founding/Senior) Engineer / Cloud Architect
* (Founding/Senior) Product / UI&UX Designer
* (Senior) Frontend Engineer
* (Mid/Junior) Full-Stack Engineer
* (Senior) 3D Artist / Motion Designer
* (Part-time, Advisory) Recruiter
* Feel free to pitch us yourself if you don't fit exactly to those roles!

Contact: apply@equitybytes.com or reply to us here!

We are exited to hear from you!

**Reply by zhyd1997** [reply]
PTAL: https://github.com/vercel/next.js/discussions/42310#discussioncomment-4030281

Apply for Frontend role.

**Reply by AsherAshrafM** [reply]
Interested in Mid/Junior Full-Stack Engineer position.

https://sachu-developer.webflow.io/

**Reply by vigneshshiv** [reply]
Hi @nwidynski, I'm interested for Full Stack Engineer role, Discussion - [#42310](https://github.com/vercel/next.js/discussions/42310#discussioncomment-4065075)
Please check out my profile - https://github.com/vigneshshiv

**Comment by chelsea-tnc** [comment]
**Tiny News Collective | Software Engineer | Fully Remote in the United States | Full-time | $115k | [Job description](https://tinynewsco.org/jobs/software_engineer)**

[The Tiny News Collective](https://tinynewsco.org/) works to make news entrepreneurship accessible and achievable to more people from a much broader range of professional, educational and lived experiences than are currently represented in our media ecosystem. We provide the tools, resources and expertise to help people build sustainable news organizations that reflect and serve their communities. We support voices historically excluded from media and media ownership.

Our tech team is growing! We're looking for a software engineer to continue to develop and maintain our complete digital news publishing system, including web publishing, newsletters, membership, ads, analytics, comments and more. We hope to expand the platform into a SaaS product.

**Position Type:** Full-time
**Compensation**: $115,000
**Location**: Fully remote, must be able to work in United States
**Team**: The engineer will report to the director of technology and work with the technology team.
**Application deadline**: Applications are reviewed on a rolling basis.

More details, including how to apply, can be found [here](https://tinynewsco.org/jobs/software_engineer).

**Comment by yaacov-commenda** [comment]
**Commenda | Software Engineer | Bangalore/Seattle/Remote | Full-time**

[Commenda](https://www.commenda.io/) is making it easier for international founders to raise money from the US. We're a back-office  as a service for startups to reduce their legal expenses and make compliance easier. 

We're an early stage company (raised preseed in July, and aiming to raise a seed round in Q2 2023) so you'd be able to have an outsized impact.

Our team is located in Bangalore and Seattle, but remote candidates will be considered.  

To get in touch, please email me at yaacov@commenda.io or comment here!

**Reply by zhyd1997** [reply]
🙌🏿 
https://github.com/vercel/next.js/discussions/42310#discussioncomment-4030281

**Reply by AsherAshraf** [reply]
Hi! I am interested in this position.
Please see my portfolio here. Thank you.
https://sachu-developer.webflow.io/

**Reply by vigneshshiv** [reply]
Hi @yaacov-commenda, I'm interested, Discussion - [#42310](https://github.com/vercel/next.js/discussions/42310#discussioncomment-4065075)
Please check out my profile - https://github.com/vigneshshiv

**Comment by rossgebhart** [comment]
**Fleetio | Web Developer | Remote (United States) | Full-time | [Apply Here](https://apply.workable.com/j/B135847BDA?utm_source=NextJS_JobBoard&utm_medium=referral)**

Fleetio is a modern software platform that helps thousands of organizations around the world manage their fleets. As a part of our growing Marketing Team, you’ll help to maintain our most important marketing asset, the marketing website - which is currently being migrated to Next.js. We're looking for a candidate who has at least 3-4 years of experience and really solid web fundamentals (css, tailwind, javascript, react, etc). 

I started as a Web Developer at Fleetio and over the last 3 years I've been promoted 4 times, so there is lots of room for growth. I am the hiring manager for this position so feel free to apply at the link above or reach out if you have questions: [rgebhart@fleetio.com](mailto:rgebhart@fleetio.com).

**Reply by uwenayoallain** [reply]
Hi @rossgebhart . I'm interested in the job, please check your email
Thanks

**Comment by HarisSpahija** [comment]
**Nederlandse Loterij | Front-end Developer | Netherlands/On-site/Hybride | Full-time | [Apply Here](https://careers.nederlandseloterij.nl/job/nederlandse-loterij/front-end-developer/?jobid=596&pubid=549)**

De Nederlandse Loterij is one of the most reliable gambling organisation in the Netherlands. It has seven strong brands. We are the market leader, experts and stand for responsible, fair and safe gambling. Almost all of our revenue goes to a happy, healthy and sporty Netherlands. In addition to being a gambling organisation with a social approach, we are an employer. We work with more than 400 top talents within our organisation.

Because we work together and win together, we always have reason to cheer. Within the Technology team, we help each other to perform and we aim for joint success! Our differences make us strong together. Movement? That's in our DNA. You and your colleagues feel at home in an environment where no day is the same.

**Location**: Netherlands / On-site / Hybride
**Relocation**: No
**Salary**: Up to €6300 ,- /month (40 hours a week)
**Stack**: Vue or React (depending on product)/Typescript/.NET/Azure
**Experience**: Min 3 years + BSc or Masters in CS or any technical field
**13th Month**: Yes
**Yearly study budget**: €2000,- free budget for a course or event related to your work

Personally I am one of the original web devs in the company and have been working exclusively here for the last two and half years (2,5). Its a really nice team and because of the nature of the business there is a lot of room to grow and learn while working with a very knowledgeable team of seniors. I highly recommend the company since the products are extremely rapid and competitive so your influence as a developer is pretty big in choosing the direction of the products. Shoot me a message if you have questions: https://www.linkedin.com/in/hspahija/

**Reply by AsherAshraf** [reply]
Hi @HarisSpahija. I'm interested in the job.

Please see my portfolio:

https://sachu-developer.webflow.io/

Thank you.

**Comment by meowlory** [comment]
STELL |  Full stack software engineer | Full-time | $100-180k base + early employee equity | [https://stell-engineering.notion.site/Founding-Engineer-full-stack-b29e391d839e4aa0a6ad18f9601579c5](url)

Hi, I'm Malory, CEO of Stell. We are a pre-seed start-up solving supply chain & engineering frictions in Aerospace & Defense. Former Hadrian & Anduril. 

We're looking for our founding software engineering team. Hiring full stack, front end web developers, and database engineers. Check out our Notion link for more info! 

We are based in Los Angeles, CA 
Due to ITAR requirements on Aerospace, US citizens or green card holders only.

malory@stell-engineering.com

**Reply by JJuggy** [reply]
Hy there, are you currently taking interns, or any junior role?

**Comment by Fullstack-dev-arch** [comment]
Location: Ukraine
Remote: Yes
Willing to relocate: Yes
Technologies: Next, React, Vue, Angular ,Ruby on Rails, Typescript, Node, Express, AWS
Resume/CV: https://drive.google.com/file/d/1afmOZ2UUQPexvmT8NiBogQZRHL7dqbYs/view?usp=share_link
Website: https://www.freelancer.com/u/SNikogosov
Email: [migumongkk@gmail.com](mailto:migumongkk@gmail.com)

**Reply by moinulmoin** [reply]
Bro, it's hiring page.

**Comment by salahawk** [comment]
Location: Remote
Technologies : Next v13, React, Tailwind CSS, Material UI

Description: I have a frontend project written in React v17, but need to convert this into Next.js v13 (which uses React v18).
Issues : Material UI v5 is compatible with React v17, but Next.js v13 uses React v18.

The task is simple:
- Project configuration (combine MaterialUI, TailwindCSS with Next.js)
- Copy/paste React code again into the repo & check if all is working fine!

**Reply by zhyd1997** [reply]
HI @salahawk 

I am looking for a job.

tell more details, which repo on github and what's your desirable DDL and budget.

**Reply by salahawk** [reply]
It's my private repo. and I'll explain in more detail on Skype.
btw, may I have your profile?

feel free to contact me on Skype : https://join.skype.com/invite/mxO32NNjcXNw

**Reply by zhyd1997** [reply]
This is my profile: https://github.com/vercel/next.js/discussions/42310#discussioncomment-4030281

but I only work for public repo...

**Reply by salahawk** [reply]
it's our companies website, cannot make it public

**Reply by zhyd1997** [reply]
DM done!

**Reply by shivamupadhyay008** [reply]
Hey, I'm also interested in the opportunity Check your dm once 
thanks!

**Reply by everest-dev0323** [reply]
Hi, @salahawk , If your position is available for me, please don't hesitate to contact me soon.
Please check my skills and experience below:
https://www.linkedin.com/in/tony-jefferson-a16355252
https://drive.google.com/file/d/1ED26yMd22Epx6m6xrr0XTMl6Rkpi7LXL/view?usp=sharing
thanks.

**Reply by paradus-hex** [reply]
Hi,

I hope you are doing well. 

Please allow me to give a brief introduction of myself. I am Yaseen Nur, a Junior React developer with full-stack M/PERN (MySQL/MongoDB/PostgreSQL,Express.js, React.js, Node.js) experience. I have used Typescript (type safety for Javascript) and Git (for version control) extensively along with frontend libraries like Material UI, Tailwind CSS, React router, React Query, and Storybook. I have also used Astro and Tailwind CSS to create static web pages.

In the back end, I worked with the Prisma ORM (for talking to the database), Zod ( for Typescript-based schema validation), and Postman (for API testing).

I have also used Python for Machine Learning and Natural Language Processing with Scikit learn, NLTK, allennlp and some other libraries.

You can find more about my skills, projects, experience, and education in my resume. I am also providing my GitHub, LinkedIn, and Portfolio website links below. They can be found on my resume as well.

I hope you will find that I am the type of competent and competitive candidate you are looking for. Please contact me at 880-170-637-3929 or via email at yaseen.nur00@gmail.com to arrange a convenient meeting time.

Thank you for your consideration, and I look forward to hearing from you soon.

-Yaseen


GitHub link: https://github.com/paradus-hex

Portfolio Website: https://paradushex.vercel.app/

LinkedIn: https://www.linkedin.com/in/yaseen-nur-taz/

**Reply by salahawk** [reply]
> DM done!

Where did you contact me?

**Comment by cheeto-bandito** [comment]
[Merkle](https://www.merkle.com/) | Sitecore Front-End Developer | US, Brazil, India - Remote | Full-time | $70k-105k US base | [Job Req](https://www.merkle.com/careers/all-openings/743999857547808)

Merkle, a Vercel [partner](https://vercel.com/experts/merkle), is seeking a talented Sitecore Frontend Developer to contribute to a dynamic team building Sitecore-based web and native app solutions within our Experience Platforms group. This role will work with our development, creative, user experience, and project management teams to develop and deploy cutting-edge websites with innovative technologies.

**Reply by Phoenix-Alpha** [reply]
@cheeto-bandito, interested in this position. Is it still relevant?

**Reply by paradus-hex** [reply]
Hi,

I hope you are doing well. 

Please allow me to give a brief introduction of myself. I am Yaseen Nur, a Junior React developer with full-stack M/PERN (MySQL/MongoDB/PostgreSQL,Express.js, React.js, Node.js) experience. I have used Typescript (type safety for Javascript) and Git (for version control) extensively along with frontend libraries like Material UI, Tailwind CSS, React router, React Query, and Storybook. I have also used Astro and Tailwind CSS to create static web pages.

In the back end, I worked with the Prisma ORM (for talking to the database), Zod ( for Typescript-based schema validation), and Postman (for API testing).

I have also used Python for Machine Learning and Natural Language Processing with Scikit learn, NLTK, allennlp and some other libraries.

You can find more about my skills, projects, experience, and education in my resume. I am also providing my GitHub, LinkedIn, and Portfolio website links below. They can be found on my resume as well.

I hope you will find that I am the type of competent and competitive candidate you are looking for. Please contact me at 880-170-637-3929 or via email at yaseen.nur00@gmail.com to arrange a convenient meeting time.

Thank you for your consideration, and I look forward to hearing from you soon.

-Yaseen


GitHub link: https://github.com/paradus-hex

Portfolio Website: https://paradushex.vercel.app/

LinkedIn: https://www.linkedin.com/in/yaseen-nur-taz/

**Comment by berpto** [comment]
sangat menarik sekali post anda, namun ada yang tau <a href="https://www.kosta-consulting.com/blog/apa-itu-erp/" target="blank">apa itu erp</a>? simak penjelasannya di https://www.kosta-consulting.com/

**Comment by evelynstender** [comment]
inploi | Front-end Engineer | London, UK | Full-time | £40k-70k | [Link](https://app.otta.com/jobs/Uy81alpx)

**About inploi**

inploi is a UK-based technology company with global ambition: rebuilding the infrastructure of online talent attraction. The labour market is in crisis, with countries all over the world grappling with the tightest hiring environment on record.

inploi’s suite of SaaS products helps enterprises to win the war for talent, empowering them to attract convert and understand the future of of their workforce.

**About the product team**

We are a group of designers, engineers, product managers, planners, analysers, gamers, skaters, and readers from 9 countries. Our obsession with technology and the freedom it brings is what get’s us out of bed in the morning. We get our kicks from building technology that solves real problems for real people all over the world and having fun whilst we do it. We have a daily sync/standup and core hours for meetings in the morning, leaving the afternoon meeting free for focus. We use Discord and Slack huddles for pairing sessions, quick chats, and hanging out.

**Reply by salahawk** [reply]
> inploi | Front-end Engineer | London, UK | Full-time | £40k-70k | [Link](https://app.otta.com/jobs/Uy81alpx)
> 
> **About inploi**
> 
> inploi is a UK-based technology company with global ambition: rebuilding the infrastructure of online talent attraction. The labour market is in crisis, with countries all over the world grappling with the tightest hiring environment on record.
> 
> inploi’s suite of SaaS products helps enterprises to win the war for talent, empowering them to attract convert and understand the future of of their workforce.
> 
> **About the product team**
> 
> We are a group of designers, engineers, product managers, planners, analysers, gamers, skaters, and readers from 9 countries. Our obsession with technology and the freedom it brings is what get’s us out of bed in the morning. We get our kicks from building technology that solves real problems for real people all over the world and having fun whilst we do it. We have a daily sync/standup and core hours for meetings in the morning, leaving the afternoon meeting free for focus. We use Discord and Slack huddles for pairing sessions, quick chats, and hanging out.

I'm interested in this role @evelynstender 
Let's connect

**Reply by everest-dev0323** [reply]
Hi, @evelynstender , If your position is available for me, please don't hesitate to contact me soon.
Please check my skills and experience below:
https://www.linkedin.com/in/tony-jefferson-a16355252
https://drive.google.com/file/d/1ED26yMd22Epx6m6xrr0XTMl6Rkpi7LXL/view?usp=sharing
thanks.

**Reply by paradus-hex** [reply]
Hi,

I hope you are doing well. 

Please allow me to give a brief introduction of myself. I am Yaseen Nur, a Junior React developer with full-stack M/PERN (MySQL/MongoDB/PostgreSQL,Express.js, React.js, Node.js) experience. I have used Typescript (type safety for Javascript) and Git (for version control) extensively along with frontend libraries like Material UI, Tailwind CSS, React router, React Query, and Storybook. I have also used Astro and Tailwind CSS to create static web pages.

In the back end, I worked with the Prisma ORM (for talking to the database), Zod ( for Typescript-based schema validation), and Postman (for API testing).

I have also used Python for Machine Learning and Natural Language Processing with Scikit learn, NLTK, allennlp and some other libraries.

You can find more about my skills, projects, experience, and education in my resume. I am also providing my GitHub, LinkedIn, and Portfolio website links below. They can be found on my resume as well.

I hope you will find that I am the type of competent and competitive candidate you are looking for. Please contact me at 880-170-637-3929 or via email at yaseen.nur00@gmail.com to arrange a convenient meeting time.

Thank you for your consideration, and I look forward to hearing from you soon.

-Yaseen


GitHub link: https://github.com/paradus-hex

Portfolio Website: https://paradushex.vercel.app/

LinkedIn: https://www.linkedin.com/in/yaseen-nur-taz/

**Reply by devknight216** [reply]
Hi, this is Jason, a UK based full stack developer. 
I noticed that you are looking for a JS fullstack developer as a team member to support your project. 
Whilst working as senior developer over the past years, I have got hands-on experience in JS frameworks, especially in libraries & frameworks such as React/Next.js, Node.js, TypeScript, Prisma, GraphQL, etc. 
For frontend working, I am good at React/Next.js, Redux, TailwindCSS, MUI, and styled components. 
Apart from that, I have built many booking, CRM web/mobile apps, familiar with integrating several services like payment gateways(Stripe, Braintree), email, sms services(Sendgrid, Twilio), and deploying to AWS(S3, EC2, Amplify, Lambda), Heroku, Atlas MongoDB by CI/CD, unit testing with Jest. 
For ticket management, I used trello/Jira/Notion. Through those opportunities, I have been familiar with team working as well, managing team project efficiently as possible.
That's why I am excited with this project, have confidence and wish a long term relationship. 
Looking forward to hearing back from you. This will be a good contract if we work together.
Thank you.

**Comment by crazydevlegend** [comment]
Looking for a Next.js v13 (familiar with React.js) who can help me complete my on-going project.
We have this completed with React, and now converting to Next.js v13.
Kindly reply if interested

**Reply by christoph724** [reply]
Hi, that sounds great! I've worked for a long time as a React developer and am very familiar with Next.js. So I'm very excited about your project. I'm confident I could step into the role and quickly get results.
So could you please discuss the project with me?
My Skype ID: live:.cid.f894bec847a3ed60

I look forward to hearing from you soon.
Thank you.

**Reply by everest-dev0323** [reply]
Hi, @crazydevlegend , If your position is available for me, please don't hesitate to contact me soon.
Please check my skills and experience below:
https://www.linkedin.com/in/tony-jefferson-a16355252
https://drive.google.com/file/d/1ED26yMd22Epx6m6xrr0XTMl6Rkpi7LXL/view?usp=sharing
thanks.

**Reply by nokibul** [reply]
Available now?

**Comment by Fullstack-dev-arch** [comment]
Hello
I am a great skilled Next.js developer who can convert your application into next
Please let me know

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by crazydevlegend** [reply]
What's your contact handle? skype.

**Comment by Fullstack-dev-arch** [comment]
live:.cid.b3bd1c5f1d397a14
This is my skype id

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Reply by salahawk** [reply]
sent

**Comment by Fullstack-dev-arch** [comment]
I sent you my skype id
Is it not working?

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by kpopper** [comment]
[Oak National Academy](https://www.thenational.academy/) | Senior Software Engineer | Remote (UK) | Part-time to Full-time | £71k | [Link](https://app.beapplied.com/apply/8eim0cq5ts?utm_source=github_nextjs)

Oak would love to hear from experienced developers who want to join a passionate, motivated team working on software the has a real and positive impact on the lives of teachers and children in the UK.

We use Typescript with Next.js for our user-facing apps and Javascript throughout our stack. We are a small but growing team working on a number of big, exciting initiatives across the organisation.

**Reply by Phoenix-Alpha** [reply]
@kpopper, do you have any opportunity for freelancers outside UK? I am interested in if you do

**Reply by kpopper** [reply]
Thanks for your interest but unfortunately we're not currently hiring outside the UK.

**Reply by christoph724** [reply]
I'm a front-end developer with rich experience in several industries. I want to join the team and learn more about the project in the team.
Please feel free to contact me.
Skype: live:.cid.f894bec847a3ed60

**Reply by crazydevlegend** [reply]
Hi, is this role still available? 
If so, kindly check my profile and DM me if interested.

Here's my:
➤ GitHub : https://github.com/crazydevlegend
➤ Portfolio : https://crazydevlegend.netlify.app/
➤ Skype : live:.cid.7117ab74680529ef
➤ Telegram : https://t.me/crazydevlegend
➤ Discord : crazydevlegend#0579

**Comment by Panguino** [comment]
BannerWave LLC | Next.js / React Developer | Remote | Contract to Full-time | Hourly Rate Dependent on Experience | [Email Us](mailto:info@bannerwave.com)


**Job Description**
We have an internal tool/software that we are going to be turning into a full SaaS platform.  The Developer is eager to digest and convert designs into a functional, eye-appealing application that provides seamless user interaction. Responsibilities include:

- Translate UI/UX design and provide end-to-end coding of applications that will produce a visually appealing and responsive product
- Implement basic dynamic controls that respond to user queries
- Advanced level understanding of CSS, JS and/or HTML programming based on assigned project
- Translate user requirements into code
- Troubleshoot application errors

**Qualifications**

- Experience with GIT and Version Control Dev Ops
- Deep understanding of CSS, JavaScript and HTML5
- Experience with Next.js, React, Redux Toolkit, GraphQL, Apollo, GSAP
- Ability to develop based on design files from Figma, Photoshop, or other design tools.
- Experience providing time estimates based on the scope of work outlined
- Minimum 3 years experience as a Developer 


**Skills**

- Ability to identify errors & inconsistencies, with strong attention to detail
- Experience working on complex Sass Platforms
- Demonstrates a keen eye for design and (visual) quality
- Ability to closely follow specified directions and adhere to project requirements
- Work proactively, independently, and utilize problem-solving skills
- Exceptional communication and interpersonal skills
- Successfully manage multiple tasks with the ability to switch gears as needed
- Provide independent critical thinking
- Navigate working with virtual teams and clients across multiple time zones

**Company Overview**
BannerWave focuses on creating interactive advertising experiences that elevate how companies and organizations educate, communicate, and engage with their market.

From strategy to production – we’re a full-service studio that creates high-end experiences through cutting-edge creative, development, and animation. We partner with brands, ad platforms, and agencies around the world to produce advertising experiences built around quality and performance.

**Reply by crazydevlegend** [reply]
Here's my Indeed profile.  
https://my.indeed.com/p/robertc-wfssl24

Kindly check and feel free to reach if interested.

**Reply by Babbili** [reply]
sent by email, thanks

**Reply by salahawk** [reply]
Kindly check your email, @Panguino .
I look forward to hearing back from you soon.

Thanks 🙇‍♂️

**Reply by Jaxenormus** [reply]
Hi, @Panguino I have sent an email inquiring about this position and attached my portfolio. I look forward to hearing back from you soon.

**Reply by Panguino** [reply]
Thanks for your interest we have found resources for this position and its now closed.

**Comment by LefterisKcode** [comment]
**Harborn** || **AWS DevOps Engineer** || **Hybrid** (in the Netherlands) || **50.000 - 70.000€  (year)** || **Full-time job** || **3 - 8 years of experience** || [Job Description](https://jobs.harborn.com/aws-devops-engineer/nl)


**_What will you bring?_**

- Hands on experience working with AWS (Networking, compute, storage & monitoring)
- Two or more years’ experience provisioning, operating, and managing complex environments
- Experience developing code in at least one high-level programming language (preferably Python)
- Experience building highly automated infrastructures
- Experience administering operating systems

{{Our tech stack consists of: _Kubernetes, Docker, Terraform, AWS EC2 with EKS, Lambda_ and _Aurora MySQL_}}

**_What do we offer?_** (some of them, because there are more)

- Laptop to work on
- Possibility of working some days from home
- Fixed compensation for working from home and your internet connection
- NS Business Card (Visa)
-  Accept Relocation
- Planning on commuting by bike? You might wanna consider, as we offer €0.40,- per kilometer
- On or around your birthday an extra day off!
- Personal development and coaching in our Academy program

Contact me on my [LinkedIn](https://www.linkedin.com/in/lefteris-koutsourakis-538b881b4/) or my email [lefteris@en-ergo.nl](lefteris@en-ergo.nl)

**Comment by AsherAshraf** [comment]
this is complete bs. asians have no chance to get hired here.

**Comment by erp-indonesia** [comment]
Pengertian <a href="https://www.kosta-consulting.com/blog/apa-itu-erp/">ERP adalah</a> sistem (perangkat lunak) yang secara otomatis mengelola proses bisnis harian, seperti finance & accounting, customer relationship management (ERP CRM) & sales, procurement, supply chain management (SCM), inventory management, dan lain sebagainya.
